bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/type/HomematicTypeGeneratorImpl.java

```diff
@@ -361,8 +354,6 @@ public ParameterOption createOption(String value, String description) {
                         }
                         builder.withMinimum(MetadataUtils.createBigDecimal(dp.getMinValue()));
                         builder.withMaximum(MetadataUtils.createBigDecimal(maxValue));
-                        builder.withStepSize(MetadataUtils
-                                .createBigDecimal(dp.isFloatType() ? Float.valueOf(0.1f) : Long.valueOf(1L)));
                         builder.withUnitLabel(MetadataUtils.getUnit(dp));
```

Comment:
One thing came to my mind: maybe we should keep step=1 for integer types.

Reply:
I guess the UI knows integers have no fraction, so it should be fine to omit it.